students/ChelseaSmith/Lesson3/mailroom.py

```diff
@@ -0,0 +1,57 @@
+record = [["James Smith", 85, 100, 75],
+          ["Alex Jones", 25, 25, 25],
+          ["Kate Smith", 100],
+          ["Kate Jones", 175, 50],
+          ["Alex Smith", 200, 200, 200]]  # creates initial donor record
+
+choose = 0  # creates variable for selecting from main menu
+
+def report(listin):  # prints a report of all donors
+    print(f"Donor name" + " " * 17 + " | Total Given | Num Gifts | Average Gift\n")
+    for row in listin:
+        sum = 0
```

Comment:
this variable is overwriting the built-in function `sum` which actually should use below ... but it would error out because of this var 😉